docs/src/architectures.md

```diff
@@ -0,0 +1,83 @@
+# Running on GPU or with MPI
+
+ClimaLand.jl is designed to run on either CPU or GPU, and to be compatible with MPI.
+This section will explain how to select the architecture you want to run your simulation on.
+
+The CliMA ecosystem uses [ClimaComms.jl](https://github.com/CliMA/ClimaComms.jl)
+to control which architecture to run on, and related internals are handled within that package.
+This isolation makes specifying the architecture to run on relatively simple.
+
+ClimaComms.jl introduces two concepts that we utilize to define the architecture
+to run a simulation on: the `device` and the `context`.
+
+The `device` specifies whether we want to run on CPU or GPU.
+The `context` contains information about the device, and also defines whether we
+want to run on an individual processor or distributedly with MPI.
+Since the device is contained within the context's data structure, when we construct
+these objects in ClimaLand we simply refer to them as the context.
+
+Below, we have descriptions for how to specify the context when trying to run a
```

Comment:
Maybe note here that the two ways are : internally in a script or in your terminal environment?

Reply:
I restructured this section to make this more clear!